frontend/src/App.js

```diff
@@ -1,37 +1,16 @@
-import axios from 'axios';
 import React from 'react';
-
-class App extends React.Component{
-    state = { details: [], }
-
-    componentDidMount() {
-        let data;
-        axios
-            .get("http://localhost:8000/api/posts/")
-            .then((res) => this.setState({ details: res.data }))
-            .catch((err) => console.log(err));
-    }
-
-    render(){
-        return(
-            <div>
-                <header>(Sunposterisk) Data Generated From Django</header>
-                <hr></hr>
-                {this.state.details.map((output, id) => (
-                    <div key={id}>
-                        <div>
-                            <h3 dangerouslySetInnerHTML={{__html: output.title}} />
-                            <p dangerouslySetInnerHTML={{__html: output.content}} />
-                            <p dangerouslySetInnerHTML={{__html: output.author}} />
-                            <p dangerouslySetInnerHTML={{__html: output.published_at}} />
-                            <p dangerouslySetInnerHTML={{__html: output.total_votes}} />
-                        </div>
-                    </div>
-                ))}
-                </div>
-        )
-    }
-
+import BlogDetail from './BlogDetail/BlogDetail';
+import { Route, Routes } from 'react-router-dom';
+import HomePage from './HomePage/HomePage';
+class App extends React.Component {
+	render() {
+		return (
+			<Routes>
+				<Route path='/' element={<HomePage />} />
+				<Route path='/blog' element={<BlogDetail />} />
```

Comment:
```suggestion
				<Route path='/blogs/:id' element={<BlogDetail />} />
```
Make your path ready for integration

`/blogs` -> all the blogs
`/blogs/:id` -> id value can be blog.id in your database. This is the identifier of which blog detail you are viewing.
Sample values:
```
/blogs/1
/blogs/5
...
/blogs/99
```